server/models/escalationPolicy.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+/**
+ * EscalationPolicy Schema
+ *
+ * rotationInterval - interval for rotating escalation policy subscribers
+ * pagingInterval - interval between sending pages to next subscriber
+ * subscribers - ordered list of users/devices to send alerts to
+ */
+const EscalationPolicySchema = new mongoose.Schema({
+  rotationInterval: {
```

Comment:
Rotation interval doesn't need to exist for an escalation policy where a user is holding devices, so this shouldn't be required.